### ISSUE TITLE
Full Balance Transfers

### ISSUE THREAD

**Issue by rwebb0418** [turtlecoin/turtle-wallet-go#12]
Got a small issue with the wallet.  The max I can transfer at any one time seems to be about 50K trtl.  This is gonna make transferring large amounts (1mil+) very time consuming, and kind of makes the "Full Balance" button pointless.    Is there anyway around this limitation?

**Comment by woodyjon** [comment]
Do you have an error message when you try to transfer more than 50K? What's the error?
At the moment I put the limit at 5 millions turtles because more than that there is a higher risk of too heavy transaction.

**Comment by rwebb0418** [comment]
Can't recall what the error was, and I'm not able to recreate it at this time.

**Comment by klesilva** [comment]
Tested with the amount of 130,000.00, as if there were a limitation of 50,000.00 per transaction
This is the error. 

**"Error Transfer.
Response with error: Transaction size is too big."**

**Comment by woodyjon** [comment]
Yes, this is known. I am working on fusion, which will solve that problem. It should be implemented within a week. In the meantime, if you cannot wait, use the CLI zedwallet, which solves that.

**Comment by klesilva** [comment]
I'll wait for the problem solved, Thank you for a response.

**Comment by woodyjon** [comment]
Can you please try the new version https://github.com/turtlecoin/turtle-wallet-go/releases/latest. When you transaction is too big, you will be proposed to make a fusion transaction. Do it and wait for that transaction to be confirmed. Then retry you big transaction. (you might have to do it multiple times). Please tell me if it works now.

side note: You might still have issues for sending your full balance, you must send a little less than your full balance. But that will normally be solved from block 620K, from which point you should be able to send your full balance.

**Comment by markveon** [comment]
It doesn’t work.  

Even if it did, its not a useable function.  Sending a million coins over and over and hoping it works is just not something that should ever be suggested or attempted.

**Comment by woodyjon** [comment]
@markveon What about telling us the version you run, your os, what you tried to do, what was the error?

We are trying to improve the tools. Not everything is perfect from the get go.

Thank you for your great and positive contribution.

**Comment by markveon** [comment]
Settle down now.  Making a GUI wallet with docker/container is awesome.  Not trying to offend.

Win 10, latest Win updates.

But this could have been tested by you.  I’m assuming you don’t want to risk your own $40 of trtl.

I was commenting on the last comment.

So I was sending a million to an address, got the fusion prompt to send to myself.  I clicked ok and waited for the tx to confirm back to my wallett.  Which was 0 trtl.

Then I tried to send back to the original destination.  Tried this at least a dozen times.

I know the work around is cli.

Sent from my iPhone

On Aug 11, 2018, at 12:15 AM, woodyjon <notifications@github.com<mailto:notifications@github.com>> wrote:


@markveon<https://github.com/markveon> What about telling us the version you run, your os, what you tried to do, what was the error?

We are trying to improve the tools. Not everything is perfect from the get go.

Thank you for your great and positive contribution.
